pkg/status/manager.go

```diff
@@ -0,0 +1,149 @@
+package status
```

Comment:
normally OSS projects undere CNCF need a license header or a SPDX entry.
is that not the case for the etcd-io org?